exercises/verify.cpp

```diff
@@ -9,10 +9,32 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+#define EXIT_SUCCESS 0
+
+bool search(int* vett, int n, int x){
+  bool trovato = 0;
+
+  for(int i=0;i<n;i++){
+    if(vett[i]==x){
+      trovato = 1;
+      break;
+    }
+  }
+  
+  return trovato;
+}
+
 int main()
 {
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int number;
+
+  cout << "Insert number: ";
+  cin >> number;
+
+  if(search(N, 10, number)==true) cout << "The number " << number << " is present in the array" << endl;
+  else cout << "The number " << number << " is not present in the array" << endl;
 
```

Comment:
ciò che cambia tra if ed else e solo quel "not" quindi dovresti creare un codice che permetta di visualizzare solo quel cambiamento, esempio:
```c++
cout << "The number " << number << " is " << search(N, 10, number) ? " " : "not " << "present in the array" << endl;
```